### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by Shubhamkr2610**
build navbar component

### CODE REVIEW

File: components/navbar/navbar.css [hunk 1]
```diff
@@ -5,43 +5,101 @@ body {
...
+
+    margin: 3rem 0rem;
+    /* box-shadow: rgb(148, 81, 81); */
 }
+
 .nav-heading {
+    
+
+    font-size: 1.25rem;
     display: inline;
```
Comment (Rockyraven) [inline components/navbar/navbar.css:29]: remove unwanted space

Comment (Shubhamkr2610) [reply components/navbar/navbar.css:29]: removed .
please review

**Review by Rockyraven — APPROVED** [review]
All goods to me go ahead

**Review by vishalg8454 — APPROVED** [review]
All fine, can proceed to merge.

**Pull request merged**